packages/core/src/anchor/anchor-processing-loop.ts

```diff
@@ -50,6 +58,10 @@ export class AnchorProcessingLoop {
       concurrency,
       store.infiniteList(batchSize),
       async (streamId) => {
+        // Exit early if we've already polled this stream in the last pacing interval
+        if (!this.checkPollTime(streamId.toString())) {
+          return
```

Comment:
let's add a 'verbose' log message here that we're skipping this, with the streamid, current time, and last poll time.